x-pack/filebeat/module/threatintel/abuseurl/test/abusechurl.ndjson.log-expected.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "@timestamp": "2019-08-10T09:02:05.000Z",
+        "event.dataset": "threatintel.abuseurl",
+        "event.module": "threatintel",
+        "fileset.name": "abuseurl",
+        "input.type": "log",
+        "log.offset": 0,
+        "service.type": "threatintel",
+        "tags": [
+            "threatintel",
+            "forwarded"
+        ],
+        "threatintel.blacklists.spamhaus_dbl": "not listed",
+        "threatintel.blacklists.surbl": "not listed",
+        "threatintel.larted": "true",
+        "threatintel.reporter": "zbetcheckin",
+        "threatintel.tags": [
+            "elf"
+        ],
+        "threatintel.threat": "malware_download",
+        "threatintel.url_status": "offline",
+        "threatintel.urlhaus_reference": "https://urlhaus.abuse.ch/url/223622/",
+        "url.domain": "45.61.49.78",
+        "url.full": "http://45.61.49.78/razor/r4z0r.mips",
```

Comment:
nit: even though this historical record is no longer malicious I would just be very very careful generally using potential maleware examples in source code for tests as someone could accidentally activate them or the tests could somehow at a later point in time.